tools/codegen/dest/lazy_ir.py

```diff
@@ -35,24 +42,55 @@ def node_ctor_arg_rvalue_string(arg: NamedCType) -> str:
         else:
             return f"{arg.name}"
 
-def node_ctor_inputs(func: LazyIrSchema) -> str:
+def node_ctor_inputs(schema: LazyIrSchema) -> str:
     """
     Produce a formatted string with the arguments as passed into the constructor of a node class.
     """
-    node_ctor_values = [node_ctor_arg_rvalue_string(arg) for arg in func.filtered_types()]
+    node_ctor_values = [node_ctor_arg_rvalue_string(arg, schema) for arg in schema.filtered_types()]
     return ",\n                              ".join(node_ctor_values)
 
+def gen_fallback_code(schema: LazyIrSchema, overload_name: str) -> str:
+    """
+    Generate code that falls back to eager conditioned on a predicate
+    """
+    fallback_args = ",\n                ".join([str(arg.name) for arg in schema.filtered_types()])
+    if len(overload_name):
+        aten_op_str = f"ATEN_OP2({schema.aten_name}, {overload_name})"
+    else:
+        aten_op_str = f"ATEN_OP({schema.aten_name})"
+    return f"""
+        if (force_eager_fallback({aten_symbol(schema)})) {{
+            return at::native::call_fallback_fn<&ltc_eager_fallback, {aten_op_str}>::call(
+                {fallback_args}
+            );
+        }}
+"""
+
+def aten_symbol(schema: LazyIrSchema) -> str:
+    missing_interned_strings = {
+        'sigmoid_backward',
```

Comment:
side note: now that this is all in core, should we just add these symbols in core instead of having to handle them specially in codegen?

Reply:
I actually added a handful of symbols to core, and was never sure why i had to do that, and ultimately figured this was a cleaner escape hatch.

if the idea is to have all the symbols be constants in core, should we just make a diff that adds all the symbols at once?  otherwise, it's not super clear why we have some and not others...

Reply:
I was trying to look into where `at::aten::{symbol}` is actually defined - do you know where it is?

 eter Bell landed some changes recently that I think are meant to codegen all of the aten symbols, so I'm wondering if this probably was basically already solved for you. (I can see `sigmoid_backward` listed in the codegen'd file `build/aten/src/ATen/core/aten_interned_strings.h`)